Base/usr/share/man/man8/useradd.md

```diff
@@ -0,0 +1,46 @@
+## Name
+
+useradd - add a new user to the system password file
+
+## Synopsis
+
+```**sh
+# useradd [options] LOGIN
+```
+
+## Description
+
+This program uses adds a new user to the system.
+
+By default, the user will be added to the `users` group.
+
+This program must be run as root.
+
+## Options
+
+* `-u` _UID_: The user identifier for the new user. If not specified, an unused UID above `1000` will be auto-generated.
+* `-g` _GID_: The group identifier for the new user. If not specified, it will default to 100 (the **users** group).
+* `-s` _SHELLBIN_: The shell binary for this login. The default is `/bin/sh`. 
+* `-m`: Create a home directory for this new user at `/home/username`.
+* `-d` _HOMEPATH_: Set the home directory for this user to _HOMEPATH).
+* `-n` _GECOS_: GECOS information about this login. See [Wikipedia](https://en.wikipedia.org/wiki/Gecos_field) for more information.
+
+## Examples
+
+```sh
+# useradd -u 300 -m kling
+# useradd -m -u 400 -g 200 -n "Sergey Bugaev" bugaevc
+# useradd quaker
+# useradd -g 1000 -d /tmp/somedir danboid
+# cat /etc/passwd
+# ls -la /home
+```
+
+## Exit Values
+
+0 - Success
+1 - Couldn't update the password file
+3 - Invalid argument to option
+4 - UID already in use
+12 - Couldn't create home directory
```

Comment:
This should be a Markdown list (use `*`s or `-`s)

And this section should be between Options and Examples

Reply:
> And this section should be between Options and Examples

Sorry I've never written a manpage before, so I'm following the layout of the Linux ones.

Reply:
Sure, follow what other systems do, but if they put exit status after examples, they're wrong :)

See Linux [`man-pages`(7)](https://linux.die.net/man/7/man-pages) for some guidance (but ignore most of the groff stuff, we use Markdown)